src/main/java/com/minecolonies/core/entity/ai/workers/crafting/AbstractEntityAICrafting.java

```diff
@@ -464,6 +464,11 @@ protected IAIState craft()
                         module.improveRecipe(currentRecipeStorage, job.getCraftCounter(), worker.getCitizenData());
                     }
 
+                    if (!currentRecipeStorage.getSecondaryOutputs().isEmpty())
+                    {
+                        building.createPickupRequest(10, currentRecipeStorage.getSecondaryOutputs());
```

Comment:
This indicates that it's not a pickup filter, but instead it's sort of a specific pickup request".